lib/screens/onboarding.dart

```diff
@@ -1,172 +1,76 @@
-                      ])),
-                  TextField(
-                      controller: _searchController,
-                      decoration: InputDecoration(
-                          border: OutlineInputBorder(
-                            borderRadius: BorderRadius.circular(10.0),
-                          ),
-                          filled: true,
-                          hintStyle: TextStyle(color: Colors.grey[700]),
-                          prefixIcon: Padding(
-                              padding:
-                                  const EdgeInsets.only(left: 16, right: 8),
-                              child: Text("twitch.tv/",
-                                  style: TextStyle(color: Colors.grey[700]))),
-                          prefixIconConstraints:
-                              const BoxConstraints(minWidth: 0, minHeight: 0),
-                          suffixIcon: GestureDetector(
-                              child: const Icon(Icons.cancel),
-                              onTap: () {
-                                _searchController.clear();
-                                setState(() {
-                                  _value = "";
-                                });
-                              }),
-                          hintText: "muxfd",
-                          fillColor: Colors.white70),
-                      style: const TextStyle(color: Colors.black),
-                      onChanged: (value) {
-                        setState(() {
-                          _value = value;
-                        });
-                      }),
-                  Flexible(
-                      child: CollapsibleWidget(
-                          expand: _value != "",
-                          child: ChannelSearchResultsWidget(
-                              query: _value,
-                              onChannelSelect: widget.onChannelSelect))),
+                  const Image(width: 160, image: AssetImage('assets/logo.png')),
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: 64),
+                      child: Text("RealtimeChat",
+                          style: Theme.of(context)
+                              .textTheme
+                              .headline6
+                              ?.copyWith(color: Colors.white))),
+                  const SizedBox(
+                    width: 400,
+                    child: Padding(
+                        padding: EdgeInsets.symmetric(horizontal: 64),
+                        child: SignInWithTwitch()),
+                  ),
+                  Text("or", style: Theme.of(context).textTheme.titleMedium),
+                  SizedBox(
+                    width: 400,
+                    child: Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 64),
+                      child: ElevatedButton(
+                        child: const Text("Continue without signing in"),
```

Comment:
Something like: or using Align
```suggestion
                        child: const Text(
                          "Continue without signing in",
                          textAlign: TextAlign.center,
                        ),
```

And I believe right now it's not horizontally centered because the property set to the parent Columns is dealing with [vertical alignment](https://docs.flutter.dev/development/ui/layout#aligning-widgets).

Reply:
Ohhh I see ok, sounds good to me! Also fixed the alignment on tablet.